src/shared/sleep-config.c

```diff
@@ -55,6 +60,7 @@ int parse_sleep_config(const char *verb, char ***_modes, char ***_states) {
                 { "Sleep",   "HibernateState",   config_parse_strv,  0, &hibernate_state },
                 { "Sleep",   "HybridSleepMode",  config_parse_strv,  0, &hybrid_mode  },
                 { "Sleep",   "HybridSleepState", config_parse_strv,  0, &hybrid_state },
+                { "Sleep",   "ValidateHibernateGenerator", config_parse_bool, 0, &validate_hibernate_generator },
```

Comment:
I struggled with a name that was in line with what the intention was about ...

If there's a change of name for the config preferred that's fine and a quick change at least that doesn't affect functional behaviour itself.